tools/update-authors.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
```

Comment:
Can you confirm the file is executable? I don't think there is a way to tell from GitHub's UI

Reply:
```
diff --git a/tools/update-authors.js b/tools/update-authors.js
new file mode 100755
```

:+1: